.evergreen/run-ocsp-responder.sh

```diff
@@ -1,10 +1,11 @@
-#! /bin/bash
-# Run an OCSP mock responder server if necessary.
+#!/usr/bin/env bash
+# Run an mock OCSP responder server if necessary.
 #
-# See the tests described in the specification for more info:
-# https://github.com/mongodb/specifications/tree/master/source/ocsp-support/tests#integration-tests-permutations-to-be-tested.
 # Precondition: mongod is NOT running. The responder should be started first.
 #
+# See the tests described in the specification for more info:
+# https://github.com/mongodb/specifications/blob/master/source/ocsp-support/tests/README.rst#integration-tests-permutations-to-be-tested
+#
 # Environment variables:
```

Comment:
Note: as mentioned in https://github.com/mongodb/mongo-c-driver/pull/1163/files#r1049184106:

>  I neglected to add `$DRIVERS_TOOLS` and `$PROJECT_DIRECTORY` and to the script's doc header as required env vars, so I'll make sure I do so before merging.